cmd/installer/pack.go

```diff
@@ -112,11 +124,19 @@ func (p *PackType) validate() error {
 	for _, file := range p.zipReader.File {
 		if filepath.Base(file.Name) == pdscFileName {
 
+			// Check if pack was compressed in a subfolder
+			subfoldersCount := strings.Count(file.Name, "/") + strings.Count(file.Name, "\\")
+			if subfoldersCount > 1 {
+				return errs.ErrPdscFileTooDeepInPack
```

Comment:
PdscFileTooDeepInPack semantically means that you found a pdsc but it is not at the right location, but actually you did not find a pdsc in the allowed locations
- root directory
- single sub directory